### ISSUE TITLE
setPageMargin() does not work

### ISSUE THREAD

**Issue by Mariovc** [Pixplicity/MultiViewPager#22]
The margins set on `setPageMargin()` method are ignored and overrided by `MultiViewPager`

**Comment by pflammertsma** [comment]
Correct; MultiViewPager adjusts the margin itself for the mechanism through
which it centers pages. I would suggest specifying the margin through the
page's layout padding instead.

**Comment by Mariovc** [comment]
Putting margin to the view is a trick to "solve" it, but that is no the correct way for two reasons:

1. The method `setPageMargin()` doesn't work
2. The margins will be animated together with the image. For example scaling the page x2 will create a double space between elements.

**Comment by pflammertsma** [comment]
Those are good points. However, I don't plan on updating this library as I instead recommend moving forward with [LinearSnapHelper](https://developer.android.com/reference/android/support/v7/widget/LinearSnapHelper.html) in conjunction with RecyclerViews.

**Comment by Mariovc** [comment]
Thanks for the point out. Could I have animations with it?

**Comment by Mariovc** [comment]
The circle view pager indicator behind the view pager is another thing that I miss from recyclerview with SnapHelper.

**Comment by Mariovc** [comment]
In the Readme there is a receference to LinearSnapHelper but I think it is not a good recplacement if you need any of the following requisites:

- Animation when turning pages.
- Bullet points. An indicator to show on which page you are.
- Pages are turned one by one without acceleration.
- Auto-scroll, pages are turned automatically.

Maybe it not a good idea to point out to LinearSnapHelper from the Readme file without taking this into account
